src/noise.jl

```diff
@@ -153,7 +154,8 @@ function simulate_noise(wf::RDWaveform, preamp::PreAmp)
     # wf values are in eV (without u"eV" units attached), noise sigma is in keV
     # if we're not simulating noise from scratch, we'll do this anyway, but noise sigma will be 0
     # -> maybe there's a better way?
-    noise_σ = ustrip(uconvert(u"eV", preamp.noise_σ))
+    noise_σ = preamp.noise_σ_keV == 0 ? preamp.noise_σ_ADC / preamp.gain : uconvert(u"eV", preamp.noise_σ_keV)
```

Comment:
According to the [definition in `preampl.jl`](https://github.com/legend-exp/LegendGeSim.jl/pull/49/files#diff-2849d998958e76bf52938428b75e136b23228359a0a267f3724380d8291369baR59), `preamp.noise_σ_keV` always has the unit `u"keV"`.
Therefore, `preamp.noise_σ_keV == 0` will never be true because the `0` is missing a unit.
This could be avoided by using `iszero(preamp.noise_σ_keV)` instead.

Reply:
You're right! Eagle eye. Fixed it.